src/containers/AdminScriptImport.jsx

```diff
@@ -52,11 +52,22 @@ export default class AdminScriptImport extends Component {
     );
 
   render() {
+    const url =
+      "https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_IMPORT_GOOGLE_DOCS_SCRIPTS_TO_IMPORT.md";
     return (
       <div>
         <CampaignFormSectionHeading
           title="Script Import"
-          subtitle="You can import interactions and canned responses from a properly formatted Google Doc."
+          subtitle={
+            <span>
+              You can import interactions and canned responses from a properly
+              formatted Google Doc. Please refer to{" "}
```

Comment:
What are the `{" "}`s surrounding the url for?

Reply:
it forces a space between `to` and the `<a href>` link -- jsx/prettier is a bit annoying here, but this is the right way to keep the space.